pom.xml

```diff
@@ -11,11 +11,11 @@
   <parent>
     <groupId>org.springframework.boot</groupId>
     <artifactId>spring-boot-starter-parent</artifactId>
-    <version>2.1.1.RELEASE</version>
+    <version>2.2.4.RELEASE</version>
   </parent>
 
   <properties>
-    <spring-boot.version>2.1.1.RELEASE</spring-boot.version>
+    <spring-boot.version>${parent.version}</spring-boot.version>
```

Comment:
Can't find this property (it's [not the default one too](https://maven.apache.org/guides/introduction/introduction-to-the-pom.html#Project_Interpolation))

Reply:
Got it, thank you!